### ISSUE TITLE
Preferred download languages for metadata only has option for Simplified Chinese

### ISSUE THREAD

**Issue by harry71019** [jellyfin/jellyfin#2241]
**Describe the bug**

When choosing preferred download language for metadatas, there is only a "Chinese" option and it only grabs metadata in Simplified Chinese (zh-CN).

**To Reproduce**

1. Go to "Edit Metadata"
2. Select "Chinese" as preferred download language for Metadata Settings
3. Save and Refresh to grab Chinese metadata
4. Note the metadata will be in Simplified Chinese (zh-CN)

**Expected behavior**

There should be a option for Traditional Chinese (zh-TW) for Taiwan and even Traditional Chinese for Hong Kong (zh-HK).

**Logs**
N/A

**Screenshots**

Using the movie _Argo_ as an example, here's what the metadata would look like when preferred download language is set as Chinese.
![image](https://user-images.githubusercontent.com/59645782/71969792-6590b980-31d5-11ea-913f-ed0f448f4c55.png)

The corresponding page for the movie in Simplified Chinese can be found at:
https://www.themoviedb.org/movie/68734-argo?language=zh-CN
which shows the same metadata
![image](https://user-images.githubusercontent.com/59645782/71969942-a7216480-31d5-11ea-8fff-5293587bc291.png)

The metadata in Traditional Chinese should be from this link:
https://www.themoviedb.org/movie/68734-argo?language=zh-TW
and will look like this
![image](https://user-images.githubusercontent.com/59645782/71970019-d041f500-31d5-11ea-9329-5060f83899e5.png)

**System (please complete the following information):**
 - OS: Windows 10 1903
 - Browser: Chrome 79.0.3945.88
 - Jellyfin Version: 10.4.3
 - Installed Plugins: none
 - Reverse proxy: none

**Additional context**
N/A

**Comment by jamiethomaswhite** [comment]
I'd like to add that setting the subtitle download language as "Chinese", selects all versions of Chinese but still prioritises and, for me at least, explicitly downloads _only_ **Simplified** Chinese.

They might still be the same language technically and even appear similar, but the difference for the user is massive. They should be treated as completely separate languages in their own right.

**Comment by dkanada** [comment]
This is a frontend bug, unfortunately a lot of the language selection elements are hardcoded but no one has taken the time to centralize the language detection.

**Comment by harry71019** [comment]
Replying to my own issue here. So I tried working on the problem but now it seems that despite we have separate settings for both "preferred download language" and "country", the code only uses the value from preferred download language.
Testing with my own server (10.7.1) and the source code from git (10.8.0), the dropdown menu now has language options for "Traditional; Chinese" and it points to the correct ISO-3166-1 tag for use in the code (zh-tw), same goes for "Traditional; Hong Kong". Jellyfin will grab the correct info from TMDB.

tldr: Just choose from the preferred download language with the correct region
